sky/provision/provisioner.py

```diff
@@ -388,21 +386,15 @@ def _post_provision_setup(
         # (3) all instances need permission to mount storage for all clouds
         # It is possible to have a "smaller" permission model, but we leave that
         # for later.
-        file_mounts = {
-            backend_utils.SKY_REMOTE_PATH + '/' + wheel_hash:
```

Comment:
could you explain a little bit about where mount wheels now?

Reply:
The mounting of the wheel is already included here:
https://github.com/skypilot-org/skypilot/blob/d38b4b4163e4d5f4ef98895e30b4b5c04fd23dde/sky/templates/aws-ray.yml.j2#L136